### ISSUE TITLE
MLX convert to gguf

### ISSUE THREAD

**Issue by AustinScola** [ggerganov/llama.cpp#10551]
### Name and Version

version: 4179 (25669aa9)
built with Apple clang version 15.0.0 (clang-1500.3.9.4) for arm64-apple-darwin23.4.0

### Operating systems

Mac

### Problem description & steps to reproduce

I'm trying to convert a lora adapter created with MLX to GGML using `convert_lora_to_gguf.py` but I'm running into a problem.

### Relevant log output

```shell
INFO:lora-to-gguf:Loading base model: llama-3.2-transformers-3b-instruct-v1
INFO:gguf.gguf_writer:gguf: This GGUF file is for Little Endian only
INFO:lora-to-gguf:Exporting model...
ERROR:lora-to-gguf:Unexpected name 'model.layers.0.self_attn.q_proj.lora_a': Not a lora_A or lora_B tensor
```

**Comment by AustinScola** [comment]
I think I may have found a solution with https://github.com/ml-explore/mlx/discussions/1507#discussioncomment-11039570 for converting from MLX to PEFT.

**Comment by ngxson** [comment]
I've never tried an MLX adapter. Could you please provide us one for testing?

**Comment by Emasoft** [comment]
This is still an issue. Also: GGUF to MLX coversion is not working.

**Comment by ngxson** [comment]
> GGUF to MLX

I don't think this is something possible. You should convert safetensors ==> MLX, not GGUF ==> MLX
